<!--
Launch file for testing purposes
-->

<launch>
  <node name="color_cloud_test" pkg="color_cloud" type="color_cloud.py" >

    <param name="pc2_topic_in"       value="pointcloud" />          <!--default = pointcloud -->
    <param name="image_topic_in"     value="usb_cam/image_raw" />   <!--default = usb_cam/image_raw -->
    <param name="pc2_topic_out"      value="rgb_cloud" />           <!--default = rgb_cloud -->

    <param name="fov_width"          value="50" />                 <!--default = 54 -->
    <param name="fov_height"         value="50" />                 <!--default = 41 -->
    <param name="width_offset"       value="+90" />                 <!--default = +90 -->
    <param name="height_offset"      value="-120" />                 <!--default = -120 --

  </node>
</launch>
